decred/decred/dcr/dcrdata.py

```diff
@@ -487,7 +487,7 @@ def __init__(self, db, params, datapath, skipConnect=False):
         self.heightMap = db.child("height", datatypes=("INTEGER", "BLOB"))
         self.headerDB = db.child("header", blobber=msgblock.BlockHeader)
         self.txBlockMap = db.child("blocklink")
-        self.tip = None
+        self.tipHeight = None
```

Comment:
Only the height is ever used, so it makes sense to store just that.

Reply:
Used here https://github.com/decred/tinydecred/blob/704e58ce332977385def797f22d2f7b4ffab6301/decred/decred/dcr/account.py#L819-L821
So could change there I guess. That's the only place I could find, but may be more.

Reply:
Fixed, thanks. Checked the codebase, no other uses.